server.js

```diff
@@ -1,39 +1,42 @@
+//-----DATA INITIALIZATION-----
 var fs = require('fs'),
   http = require('http');
-
+  
 var dane = {},
   port = 801;
-dane['school_name'] = "ZS1 Bochnia";
-dane['school_ico'] = "http://zs1.bochnia.pl/images/logobiale.png";
+dane.school_name = "ZS1 Bochnia";

```

Comment:
to nie zadziala niestety https://stackoverflow.com/questions/12953704/how-to-access-object-properties-containing-special-characters